composer.json

```diff
@@ -6,17 +6,17 @@
     "license": "MIT",
     "require": {
         "php": ">=7.2",
-        "cakephp/cakephp": "~4.1",
+        "cakephp/cakephp": "~4.1.0",
         "cakephp/migrations": "^3.0",
         "cakephp/plugin-installer": "^1.2",
         "mobiledetect/mobiledetectlib": "^2.8"
     },
     "require-dev": {
         "cakephp/bake": "^2.0.3",
-        "cakephp/cakephp-codesniffer": "~4.2",
+        "cakephp/cakephp-codesniffer": "~4.2.0",
         "cakephp/debug_kit": "^4.1",
         "josegonzalez/dotenv": "^3.2",
-        "phpunit/phpunit": "^8.5",
+        "phpunit/phpunit": "~8.5.0",
```

Comment:
There was no need to limit those require-dev IMO.

Reply:
I synced it with core.